### PR TITLE
Add statistics page

### LINKED ISSUE #38: Feature: statistics page
Would be nice to have a page with statistics

### PR DESCRIPTION
**Pull request by KiOui**
Add a statistics page for TOSTI.

Closes #38

### CODE REVIEW

File: website/tosti/settings/base.py [hunk 1]
```diff
@@ -190,10 +190,11 @@
     'THALIEDJE_START_PLAYERS_AT': ("08:00", 'Time to start the players (only on weekdays). Should be aligned on 5 minutes', str),
     'THALIEDJE_HOLIDAY_ACTIVE': (False, 'If enabled, the player will not start playing automatically at the start of the day', bool),
     'THALIEDJE_START_PLAYER_URI': ('', 'URI to start playing when the player starts automatically.', str),
+    'STATISTICS_BEER_ID': (0, 'The object ID of the "Beer" product in the Borrel application', int),
```
Comment (JobDoesburg) [inline website/tosti/settings/base.py:193]: Maybe this should be a category, as there are multiple kinds of beer

Comment (KiOui) [reply website/tosti/settings/base.py:193]: Haha that was really quick, I thought about that but then we also include 0.0 beer, I don't know if we want that? Let's leave it like this for now, we can always change this later.

Comment (JobDoesburg) [reply website/tosti/settings/base.py:193]: we currently have the following

  | Hoegaarden Radler 0.0 | Enter the amount of bottles, not the amount of crates. Volume: 0.25L Alcoholpercentage: 0.04% (This is less then the alcoholpercentage of the air in the canteens) | 0.50 | Beer |  
-- | -- | -- | -- | -- | --
  | Enter the amount of bottles. Remember: one crate fits 24 bottles. Volume: 0.33L Alcoholpercentage: 8.4% | 1.71 | Beer |  
  | Enter the amount of bottles. Remember: one crate fits 24 bottles. Volume: 0.33L Alcoholpercentage: 6.6% | 1.42 | Beer |  
  | Enter the amount of bottles. Remember: one crate fits 24 bottles. Volume: 0.25L Alcoholpercentage: 4.1% | 1.19 | Beer |  
 

[Hoegaarden Radler 0.0](https://tosti.science.ru.nl/admin/borrel/product/27/change/?_changelist_filters=category__id__exact%3D1)	Enter the amount of bottles, not the amount of crates. Volume: 0.25L Alcoholpercentage: 0.04% (This is less then the alcoholpercentage of the air in the canteens)	0.50	Beer	True
	[Tripel Karmeliet](https://tosti.science.ru.nl/admin/borrel/product/22/change/?_changelist_filters=category__id__exact%3D1)	Enter the amount of bottles. Remember: one crate fits 24 bottles. Volume: 0.33L Alcoholpercentage: 8.4%	1.71	Beer	True
	[Leffe Blond](https://tosti.science.ru.nl/admin/borrel/product/21/change/?_changelist_filters=category__id__exact%3D1)	Enter the amount of bottles. Remember: one crate fits 24 bottles. Volume: 0.33L Alcoholpercentage: 6.6%	1.42	Beer	True
	[Belle-Vue Extra Kriek](https://tosti.science.ru.nl/admin/borrel/product/14/change/?_changelist_filters=category__id__exact%3D1)	Enter the amount of bottles. Remember: one crate fits 24 bottles. Volume: 0.25L Alcoholpercentage: 4.1%	1.19	Beer	True
	[Hertog Jan](https://tosti.science.ru.nl/admin/borrel/product/1/change/?_changelist_filters=category__id__exact%3D1)

Comment (JobDoesburg) [reply website/tosti/settings/base.py:193]: We currently have 5 types of beer. Yes it does include 0.0 but I do think it is best to have the whole category actually

**Review by JobDoesburg — APPROVED** [review]

**Pull request merged**